src/genome.py

```diff
@@ -109,7 +112,6 @@ def createInstallationStep(self):
         *        their versions/commits and whether they are installed or not
         * @returns zero on success
         '''
-        '''
         systemState = self.systemGoal.clone()
         self.upgradeStep.append(SystemState)
```

Comment:
I expect this is a bug. SystemState suggests the class we removed. If it refers to the line above it, then lowercase s, i.e. systemState